settings.py

```diff
@@ -1351,11 +1356,11 @@
 
 CUSTOM_UCR_EXPRESSIONS = [
     ('abt_supervisor', 'custom.abt.reports.expressions.abt_supervisor_expression'),
-    ('mvp_medical_cause', 'custom.mvp_ucr.reports.expressions.medical_cause_expression'),
-    ('mvp_no_treatment_reason', 'custom.mvp_ucr.reports.expressions.no_treatment_reason_expression'),
-    ('mvp_treatment_provider_name', 'custom.mvp_ucr.reports.expressions.treatment_provider_name_expression'),
-    ('mvp_treatment_place_name', 'custom.mvp_ucr.reports.expressions.treatment_place_name_expression'),
-    ('mvp_death_place', 'custom.mvp_ucr.reports.expressions.death_place_expression'),
+    ('mvp_medical_cause', 'mvp.ucr.reports.expressions.medical_cause_expression'),
```

Comment:
that directory gets automatically added to the path (https://github.com/dimagi/commcare-hq/blob/master/manage.py#L43)